src/javascript/static/pages/regulation.js

```diff
@@ -21,6 +24,13 @@ const Regulation = (() => {
                     relocateLinks();
                 }
             });
+
+            BinarySocket.wait('website_status').then(() => {
+                if (isFrance()) {
+                    const $cfd_selector = $('#cfd_fillbox');
+                    $cfd_selector.remove();
```

Comment:
better to use javascript if jquery is not necessary

Reply:
Got it. Fixed that and simplified in latest commit.